### DISCUSSION TITLE
Shouldn't WeatherForecast belong in Domain?

### DISCUSSION THREAD

**Discussion by mathew-odwyer** [jasontaylordev/CleanArchitecture#548] [Q&A] [answered]
Isn't [WeatherForecast](https://github.com/jasontaylordev/CleanArchitecture/blob/main/src/Application/WeatherForecasts/Queries/GetWeatherForecasts/WeatherForecast.cs) considered a domain model?

**Comment by wahidrezgui** [comment, marked as answer]
Hy, YES I suppose if you want to integrate it in your Business layer you can create the domain entity and integrate the business rules in your application layer . I think it is still there now because it's a sample created by default when the project was created.
